dart/dynamics/FreeJoint.cpp

```diff
@@ -58,14 +58,32 @@ FreeJoint::~FreeJoint()
 }
 
 //==============================================================================
+Eigen::Vector6d FreeJoint::convertToPositions(const Eigen::Isometry3d& _tf)
+{
+  Eigen::Vector6d x;
+  x.head<3>() = math::logMap(_tf.rotation());
```

Comment:
Let's use linear() consistently to see more clear if the inaccuracy causes problem or not.